css/styles.css

```diff
@@ -0,0 +1,524 @@
+:root {
+    box-sizing: border-box;
+    color: black;
+    font-size: 62.5%;
+    margin: 0;
+    scrollbar-width: none;
+    -ms-overflow-style: none;
+}
+
+/* Hide y-scrollbar in Chrome & Safari */
+::-webkit-scrollbar { 
+    display: none; 
+    width: 0 !important;
+}
+  
+/* GENERAL STYLES */
+
+li {
+    list-style: none; 
+    display: inline-block;
+}
+
+/* LAYOUT */
+
+.inline {
+    display: inline-block;
+}
+  
+.block {
+    display: block; 
+}
+
+.sticky {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 100;
+}
+  
+@media only screen and (max-width: 767px) {
+    .hidden-md-down {
+        display: none; 
+    } 
+}
+
+@media only screen and (min-width: 1400px) {
+    body {
+        max-width: 1400px;
+        margin: 0 auto;
+    }
+}
+
+
+/* TYPOGRAPHY */
+
+@font-face {
+    font-family: 'Anton';
+    src: url("../assets/fonts/Anton/Anton-Regular.ttf");
+}
+
+@font-face {
+    font-family: 'EB Garamond';
+    src: url("../assets/fonts/EB_Garamond/EBGaramond-Regular.ttf");
+}
+
+@font-face {
+    font-family: 'Montserrat';
+    src: url("../assets/fonts/Montserrat/Montserrat-Regular.ttf");
+}
+
+h1, .work-card .description h1 {
+    font-family: 'Anton', sans-serif;
+    font-size: 7rem;
+    letter-spacing: 0.2rem;
+}
+
+h2, .highlight-text {
+    font-size: 3rem;
+    font-style: italic;
+    font-family: 'EB Garamond', serif;
+}
+
+p, .work-card .description h2 {
```

Comment:
`p` text looks a little large on small screen sizes, maybe use a media query to scale font

Reply:
will consider this too 👍